container-templates/docker-compose/.devcontainer/docker-compose.yml

```diff
@@ -23,6 +23,15 @@ services:
       # This lets you avoid setting up Git again in the container
       - ~/.gitconfig:/root/.gitconfig
 
-    # Overrides default command so things don't shut down after the process ends - useful for debugging
+      # Uncomment the next line if you plan to install the Docker CLI. See the docker-in-docker-compose definition for details.
+      # - /var/run/docker.sock:/var/run/docker.sock 
+
+    # Uncomment the next four lines if you will use a ptrace-based debugger like C++, Go, and Rust.
+    # cap_add:
+    #   - SYS_PTRACE
+    # security_opt:
+    #   - seccomp:unconfined
+
+    # Overrides default command so things don't shut down after the process ends - useful for debugging.
```

Comment:
I'd drop 'useful for debugging', the container must not shutdown for any development activity.